pjsip/src/pjsua-lib/pjsua_acc.c

```diff
@@ -370,7 +370,9 @@ static pj_status_t initialize_acc(unsigned acc_id)
         }
         pj_list_push_back(&acc->route_set, r);
     }
-
+       pj_list_push_back(&acc->route_set, pjsip_hdr_shallow_clone(acc->pool, pjsua_var.outbound_proxy.next));
+       pj_list_push_back(&acc->route_set, pjsip_hdr_shallow_clone(acc->pool, pjsua_var.outbound_proxy.prev));
+ 
```

Comment:
Adding first & last global route to acc `route_set` list, what for? Perhaps also add some comments?